Editors/README.md

```diff
@@ -72,6 +72,37 @@ You will need the path to the `sourcekit-lsp` executable and the Swift toolchain
 
 There is an Emacs client for SourceKit-LSP in the [main Emacs LSP repository](https://github.com/emacs-lsp/lsp-sourcekit).
 
+## Vim 8
+
+Install [vim-lsp](https://github.com/prabirshrestha/vim-lsp). In your `.vimrc`, configure vim-lsp to use
+sourcekit-lsp for Swift source files like so:
+
+```
+if executable('sourcekit-lsp')
+    au User lsp_setup call lsp#register_server({
+        \ 'name': 'sourcekit-lsp',
+        \ 'cmd': {server_info->['sourcekit-lsp']},
+        \ 'whitelist': ['swift'],
+        \ })
+endif
+```
+
+(…assuming `sourckit-lsp` is in your PATH variable, otherwise replace `'sourcekit-lsp'` with path to your
+command location).
+
+That's it! As a test, open a source file in an Xcode project, put cursor on top of a symbol in normal mode and
```

Comment:
Let's say something like "open a swift file" and not mention Xcode.  We don't directly support Xcode project settings anyway at this point.